cart-page/cart.html

```diff
@@ -0,0 +1,170 @@
+                        <h1 class="card-title">RTX 2080Ti<i class="fas fa-heart"></i></i></h1>
+                        <h2 class="card-subtitle">Nvidia</h2>
+                        <p class="card-description">Your Text</p>
+                        <div class="search-categories">
+                            <p>Quantity:</p>
+                            <div class="select">
+                                <select name="search-categories" id="search-categories">
+                                    <option value="0">0 (Delete)</option>
+                                    <option value="1" selected="selected">1</option>
+                                    <option value="2">2</option>
+                                    <option value="3">3</option>
+                                    <option value="4">4</option>
+                                    <option value="5">5</option>
+                                </select>
+                            </div>
+                        </div>
+                        <div class="card-btn">
+                            <button class="btn">Remove from Cart</button>
+                        </div>
+                    </div>
+                    <div class="card-container product-card card-shadow">
+                        <img class="card-img" src="https://picsum.photos/400/400" alt="..." />
+                        <h1 class="card-title">RTX 2080Ti<i class="fas fa-heart"></i></i></h1>
+                        <h2 class="card-subtitle">Nvidia</h2>
+                        <p class="card-description">Your Text</p>
+                        <div class="search-categories">
+                            <p>Quantity:</p>
+                            <div class="select">
+                                <select name="search-categories" id="search-categories">
+                                    <option value="0">0 (Delete)</option>
+                                    <option value="1" selected="selected">1</option>
+                                    <option value="2">2</option>
+                                    <option value="3">3</option>
+                                    <option value="4">4</option>
+                                    <option value="5">5</option>
+                                </select>
+                            </div>
+                        </div>
+                        <div class="card-btn">
+                            <button class="btn">Remove from Cart</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="ammount-container stick">
+                    <div class="card-container card-shadow">
+                        <h2 class="card-subtitle bold">Price Detail</h2>
+                        <div class="filter-divider-line"></div>
+                        <h2 class="card-subtitle">Price: <span>90000Rs</span></h2>
+                        <h2 class="card-subtitle">Quantity: <span>3</span></h2>
```

Comment:
Not related to development, but `Rs` should not be written after the amount. It should always be `Rs. 90000`. Or use `₹` before the amount

Reply:
thanks for that apurva I will change it around